lib/dotinstall/util.py

```diff
@@ -115,6 +115,39 @@ def create_symlink(source: str, target: str) -> None:
     os.symlink(source, target)
 
 
+def concat_files(
+    sources: T.List[str], target: str, comment_fmt: T.Optional[str] = None
+) -> None:
+    sources = [os.path.abspath(os.path.expanduser(path)) for path in sources]
+    target = os.path.expanduser(target)
+    _remove_symlink(target)
+    create_dir(os.path.dirname(target))
+    logger.info("Concatenating %s to %s...", sources, target)
+    with open(target, "w") as target_handle:
+        if comment_fmt:
+            print(comment_fmt % "DO NOT EDIT THIS FILE", file=target_handle)
+            print(
+                comment_fmt
+                % "This file was concatenated by dotfiles utility.",
+                file=target_handle,
+            )
+            print(
+                comment_fmt % "All changes in this file will not be retained.",
+                file=target_handle,
+            )
+            print(file=target_handle)
+        for source in sources:
```

Comment:
I'd put a blank line before that `for` loop

Reply:
Not valid with Black
https://github.com/rr-/dotfiles/pull/13/checks?check_run_id=1215213028

Reply:
Oh, nevermind. Those were empty spaces!